src/main/cluster.ts

```diff
@@ -142,6 +161,7 @@ export class Cluster implements ClusterModel {
     await this.refreshConnectionStatus()
     if (this.accessible) {
       await this.refreshAllowedResources()
+      this.isAdmin = await this.isClusterAdmin()
```

Comment:
This has got lost at some point during refactoring, we might want to backport this to `master`.